aten/src/ATen/native/native_functions.yaml

```diff
@@ -13478,6 +13478,173 @@
     CompositeExplicitAutograd: special_zeta_out
   tags: pointwise
 
+- func: special_betainc(Tensor self, Tensor a, Tensor b) -> Tensor
+  device_check: NoCheck   # TensorIterator
+  python_module: special
+  variants: function
+  structured_delegate: special_betainc.out
+
+- func: special_betainc.x_scalar(Scalar self, Tensor a, Tensor b) -> Tensor
```

Comment:
Do we really need the scalar overloads for these?
We would wrap python numbers into Tensors automatically and that would mean we only need the Tensor/Tensor/Tensor overload above.

Reply:
I wrote the scalar overloads, to correspond with @nicola-decao 's code.
I'll change the scalar overloads to wrapped python numbers, as you suggested.